UPGRADE-1.9.md

```diff
@@ -76,6 +48,52 @@
     +           permanent: false
     ```
 
+#### Upgrades Symfony to v5.2
+
+1. Upgrade Symfony dependencies by:
+
+    ```bash
+    composer config extra.symfony.require "^5.2"
+    composer require "symfony/browser-kit" "^5.2"
+    composer require "symfony/debug-bundle" "^5.2"
+    composer require "symfony/intl" "^5.2"
+    composer require "symfony/web-profiler-bundle" "^5.2"
+    ```
+
+1. Add new bundles to your list of used bundles in `config/bundles.php` if they are not already there:
+
+    ```diff
+    +   BabDev\PagerfantaBundle\BabDevPagerfantaBundle::class => ['all' => true],
+    +   SyliusLabs\Polyfill\Symfony\Security\Bundle\SyliusLabsPolyfillSymfonySecurityBundle::class => ['all' => true],
+
+1. Remove `getContainerLoader` method from `src/Kernel.php` class if you did not customise it.
+
+1. We've removed the support for Symfony's Templating component (which is removed in Symfony 5). 
+
+    * Remove `templating` from framework's configuration:
+        
+        ```diff
+        # config/packages/framework.yaml
+       
+        framework:
+            # ...
+        -    templating: { engines: ["twig"] }
+        ```
+      
+    * Replace any usages of `Symfony\Bundle\FrameworkBundle\Templating\EngineInterface` with `Twig\Environment`.
+   
+        Inject `twig` service into your controllers instead of `templating`.
```

Comment:
```suggestion
        Inject `twig` service into your controllers instead of `templating` or `templating.engine.twig`.
```

We used this second service in RefundPlugin.